test/functional/apps/management/_scripted_fields.js

```diff
@@ -97,7 +97,7 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.discover.clickFieldListItemVisualize(scriptedExpressionFieldName);
         await PageObjects.header.waitUntilLoadingHasFinished();
         await PageObjects.visualize.waitForVisualization();
-        await PageObjects.visualize.collapseChart();
+        await PageObjects.visualize.toggleSpyPane();
```

Comment:
Pane was intentional, though I don't know which is more correct.  Looks like I'm not the only one who uses the two terms somewhat interchangeably - http://www.techwr-l.com/archives/9811/techwhirl-9811-00780.html#.WVUWWxPyvm0

But a quick search of our code and looks like `panel` is more consistent, so I'll switch it!

<img width="671" alt="screen shot 2017-06-29 at 11 02 57 am" src="https://user-images.githubusercontent.com/16563603/27694609-8f8bf904-5cba-11e7-9aa0-3a12495cec2f.png">